test/classes/example/tasks/TestEcho.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
```

Comment:
Would you mind changing the file comments to `/**` and omit the empty line 2.

Reply:
fixed

Reply:
> 
> 
> fixed

no

Reply:
I found more files with this type of doc block and fixed them all. I added this doc block to files where it was missing.